From d67877118f61a47a2875bd12e7f9d1f14833b770 Mon Sep 17 00:00:00 2001
From: Yusuke Mitsuki <mickey.happygolucky@gmail.com>
Date: Sat, 18 Feb 2023 11:53:02 +0900
Subject: [PATCH 2/2] eth: modify phy address

---
 include/configs/imx8mp_evk.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/imx8mp_evk.h b/include/configs/imx8mp_evk.h
index 4a69e275e6..fab03f6353 100644
--- a/include/configs/imx8mp_evk.h
+++ b/include/configs/imx8mp_evk.h
@@ -51,9 +51,9 @@
 #define CONFIG_ETHPRIME                 "eth1" /* Set eqos to primary since we use its MDIO */
 
 #define CONFIG_FEC_XCV_TYPE             RGMII
-#define CONFIG_FEC_MXC_PHYADDR          1
+#define CONFIG_FEC_MXC_PHYADDR          0
 
-#define DWC_NET_PHYADDR			1
+#define DWC_NET_PHYADDR			0
 
 #define PHY_ANEG_TIMEOUT 20000
 
-- 
2.25.1

